Sources/Automerge/patch.swift

```diff
@@ -4,10 +4,18 @@ import enum AutomergeUniffi.PatchAction
 typealias FfiPatchAction = AutomergeUniffi.PatchAction
 typealias FfiPatch = AutomergeUniffi.Patch
 
+/// A collection of changes to an Automerge document.
```

Comment:
I think this is a bit confusing because we also refer to change hashes elsewhere, but the "change" in "change hash" is not the same as the change in a patch. (This is a symptom of some general muddiness around the design of the patch API I think)

A patch really represents a change in the observable state, rather than any change at all. It's intended to be used for applications to observe when something has happened that they might need to respond to.

Maybe a better name would be `Diff`?

In any case I don't think we should refer to them as "changes", just to avoid confusion.

Reply:
Ah! Very good - I wasn't entirely sure and hadn't re-read all the code to know, so I was guessing based on the name alone. I'll rip out the "change" phrasing here entirely. I'll re-use the detail you just provided - and I think the name is a lot more relevant if you envision an analogue to "diff/patch" in the Git/source-control world.